mappings/net/minecraft/class_5315.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5315
+	FIELD field_25024 version I
```

Comment:
Might specify this is the version of level.dat

Reply:
What do you mean exactly?

Reply:
this is the level.dat format version, not the data version of the save represented by the level.dat